electron-builder.json

```diff
@@ -1,30 +1,21 @@
-/**
- * @see https://www.electron.build/configuration/configuration
- */
 {
   "$schema": "https://raw.githubusercontent.com/electron-userland/electron-builder/master/packages/app-builder-lib/scheme.json",
   "appId": "com.WakaTime.WakaTime",
   "asar": true,
-  "productName": "Wakatime",
-
+  "publish": null,
+  "productName": "WakaTime Desktop",
```

Comment:
Is this name visible to users? Then it should be just `WakaTime`

Reply:
renamed back to WakaTime